charts/incubator/kubernetes-reflector/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+env:
+  ES_Serilog__MinimumLevel__Default: Information
+  ES_Reflector__Watcher__Timeout: ""
+  ES_Reflector__Kubernetes__SkipTlsVerify: "false"
+
+operator:
+  register: true
```

Comment:
I don't remember if we need any changes on common lib side for this.
Also, is this going to be a required operator in order other app to work?

Not sure if we should register it, if its optional, we can revisit this later.

Reply:
Maybe. The `values.yaml` of the common chart have some config holders:

https://github.com/truecharts/library-charts/blob/a7cdda71c581c0f4cba37890dddcf09d2257f51c/library/common/values.yaml#L588-L607

Reply:
It will eventually be needed by clusterIssuer in order to replicate the certificates it generates in its own namespace.

Reply:
But I can remove it for now and add a patch (please provide some guidance on what is needed)  to Common if you want.

Reply:
I'll check and get back to you